barretenberg/cpp/src/barretenberg/vm2/constraining/relations/gas.test.cpp

```diff
@@ -102,5 +105,27 @@ TEST(GasConstrainingTest, GasUsedContinuity)
     EXPECT_THROW_WITH_MESSAGE(check_relation<gas>(trace, gas::SR_DA_GAS_USED_CONTINUITY), "DA_GAS_USED_CONTINUITY");
 }
 
+TEST(GasConstrainingTest, DynGasFactorBitwise)
```

Comment:
@fcarreiro is having the test here fine? Is there a relation i can specifically test to check that the `previous_gas + this_gas = gas_used`?